lms/templates/header/navbar-authenticated.html

```diff
@@ -57,7 +58,11 @@
       </div>
     % endif
     <div class="mobile-nav-item hidden-mobile nav-item">
-      <a class="help-link" href="${get_online_help_info(online_help_token)['doc_url']}" target="_blank">${_("Help")}</a>
+      % if online_help_token == "instructor":
+        <a class="help-link" href="${get_online_help_info(online_help_token)['doc_url']}" target="_blank">${_("Help")}</a>
+      % else:
+        <a class="help-link" href="${configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK)}">${_("Help")}</a>
```

Comment:
We can come back in a future ticket to add context-sensitivity here, adding a new ['help_center_url'] parameter (similar conceptually to the current doc_url) cc'ed @catong 

Once we have an updated Help Center in February we can revisit adding page tokens for specific help center article links.

Reply:
Done